### PR TITLE
Support for RAP domains.

### PR DESCRIPTION
**Pull request by christinaholtNOAA**
This adds the necessary changes for creating RAP maps.

- Template the NCL variable names to include grid definition (retrieved from first variable entry of file).
- Support Lambert conformal, stereographic, and rotated lat/lon.
- Define single set of sub domains used by both RAP and HRRR.
- Move add_logo from create_graphics.py to figures maps.py to put it consistently in same place for all domains.
- Define wind stride based on domain.
- Update image_lists/hrrr_subset.yml, and alphabetize image lists.

This parallel version will not support timely generation of all graphics in a single job. More work is needed to support that effort.

Passes tests/linter.

### CODE REVIEW

File: adb_graphics/datahandler/grib.py [hunk 1]
```diff
@@ -138,6 +146,11 @@ def get_level(self, field, level, spec):
 
         ''' Returns the value of the level to for a 3D array '''
 
+        # The index of the requested level
+        lev = spec.get('vertical_index')
+        if lev is not None:
+            return lev
+
```
Comment (christinaholtNOAA) [inline adb_graphics/datahandler/grib.py:153]: Stop doing work here, if vertical index is set explicitly.

**Comment by Brian-Jamison** [comment]
@christinaholtNOAA why was I removed as a reviewer?

**Review by cshartsough — APPROVED** [review]
I could follow most of what you did, so I'm getting better.  ;-)

Didn't see anything off.  A lot of nice work here.

**Comment by christinaholtNOAA** [comment]
@Brian-Jamison I'm not sure. Maybe a random stray click? It wasn't intentional.

**Comment by cshartsough** [comment]
I've had time strange times choosing reviewers, too, having to click and re-click to get it right.  Something odd with GitHub, I guess.

**Comment by Brian-Jamison** [comment]
Ok, I assumed it was unintentional, though Craig is a better reviewer, TBH.
Great work!  I am also understanding more as I go along.

**Review by Brian-Jamison — APPROVED** [review]
Awesome sauce!

**Pull request merged**